src/services/user/hooks/registrationPins.js

```diff
@@ -7,12 +7,13 @@ const {
 } = require('../../../../config/globals');
 const globalHooks = require('../../../hooks');
 const pinModel = require('../model').registrationPinModel;
+const { getRandomInt } = require('../../../utils/randomNumberGenerator');
 
 const removeOldPins = (hook) => pinModel.deleteMany({ email: hook.data.email })
 	.then(() => Promise.resolve(hook));
 
 const generatePin = (hook) => {
-	const pin = Math.floor((Math.random() * 8999) + 1000);
+	const pin = getRandomInt(9998, 1000);
```

Comment:
Can you please modified it to 9999. It is a valid pin too.

Reply:
yeah sure. I have reproduced it as it was before this code has never generated 9999 max was 9998
![image](https://user-images.githubusercontent.com/5371568/81271547-8ca8ac00-904c-11ea-945c-064e48c24227.png)

Reply:
done :+1:

Reply:
Ok that it is fixed now, thank you for it. :)